woocommerce/woocommerce-android
# Swipe to complete analytics

Closes: #7038 

### Description
This PR updates the `ORDER_STATUS_CHANGE` event to be able to measure the adoption of the new Swipe-To-Complete feature.

### Testing instructions

1. Open the orders tab
2. Swipe to complete a non-completed order
3. Check that an `order_status_change` event is fired and the `flow` property is `list`

### Images/gif

<img width="1638" alt="Screen Shot 2022-08-03 at 16 57 07" src="https://user-images.githubusercontent.com/18119390/182697766-44ad7c0d-50d1-48b1-8d6f-a616c317669d.png">
